sdks/java/extensions/sql/udf-test-provider/build.gradle

```diff
@@ -30,5 +30,5 @@ project.ext.jarPath = jar.archivePath
 dependencies {
   // No dependency (direct or transitive) on :sdks:java:core.
   compile project(":sdks:java:extensions:sql:udf")
-  compile library.java.guava
+  compile library.java.vendored_guava_26_0_jre
```

Comment:
@ibzib, @apilloud I assume we are intentionally not using the vendored guava here so we don't have beam dependencies?

Should the fix be to add a checkstyle exception for this directory?

Reply:
This is a test, so I think it can safely depend on either. It might be easiest to just remove the guava dependency.